### PR TITLE
Add HTML, CSS, and JS files

### PR DESCRIPTION
**Pull request by c00p75**
- Added HTML file
- Added CSS file
- Added JavaScript files

### CODE REVIEW

File: Modules/Book HTML.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export default function appendBookHTML(title, author, element) {
+  const newBook = document.createElement('div');
+  newBook.setAttribute('class', 'book-list');
+  const newEntry = document.createElement('div');
+  newEntry.innerHTML = `<span> "${title}" by ${author} </span><button class="remove">Remove</button>`;
+  newBook.appendChild(newEntry);
+  element.appendChild(newBook);
+  return false;
+}
```
Comment (Bondok6) [inline Modules/Book HTML.js]: - [ ] According to the requirements, you should refactor your methods and functions to arrow functions, so Kindly update it 👍

image

File: Modules/Book update.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import appendBookHTML from './Book HTML.js';
+// eslint-disable-next-line max-len
+export default function updateNewBook(array, element) {
+  appendBookHTML(array[array.length - 1].title, array[array.length - 1].author, element);
+  localStorage.setItem('Awesome Book Collection', JSON.stringify(array));
+}
```
Comment (Bondok6) [inline Modules/Book update.js]: - [ ] The same thing here, kindly check my comment above ⬆

File: Modules/Clear fields.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export default function clear(title, author) {
+  title.value = '';
+  author.value = '';
+  return true;
+}
```
Comment (Bondok6) [inline Modules/Clear fields.js]: - [ ] The same thing here, kindly check my comment above ⬆

File: Modules/Remove book.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export default function removeBook(bookElement, collection, array) {
+  const parent = bookElement.parentElement.parentElement;
+  const parrentIndex = Array.from(collection.children).indexOf(parent);
+  parent.remove();
+  array.splice(parrentIndex, 1);
+  localStorage.setItem('Awesome Book Collection', JSON.stringify(array));
+}
```
Comment (Bondok6) [inline Modules/Remove book.js]: - [ ] The same thing here, kindly check my comment above ⬆

File: Modules/Retrieve data.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import appendBookHTML from './Book HTML.js';
+
+export default function loadBooks(array, element) {
+  const stored = localStorage.getItem('Awesome Book Collection');
+  if (stored !== null) { array = JSON.parse(stored); }
+  array.forEach((book) => appendBookHTML(book.title, book.author, element));
+}
```
Comment (Bondok6) [inline Modules/Retrieve data.js]: - [ ] The same thing here, kindly check my comment above ⬆

File: Modules/Section display.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+export function contactSection() {
+  document.getElementById('contact').style.display = 'block';
+  document.getElementById('book-list').style.display = 'none';
+  document.getElementById('add-book').style.display = 'none';
+  document.getElementById('list').style.color = 'Black';
+  document.getElementById('contact-btn').style.color = 'Blue';
+  document.getElementById('add').style.color = 'Black';
+}
```
Comment (Bondok6) [inline Modules/Section display.js]: - [ ] The same thing here, kindly check my comment above ⬆

File: Modules/Validate.js [hunk 7]
```diff
@@ -0,0 +1,4 @@
+export default function validation(titleField, authorField) {
+  if (titleField.value === '' || authorField.value === '') { return false; }
+  return true;
+}
```
Comment (Bondok6) [inline Modules/Validate.js]: - [ ] The same thing here, kindly check my comment above ⬆

File: Modules/date time.js [hunk 8]
```diff
@@ -0,0 +1,4 @@
+export default function setDateTime(element) {
+  const date = new Date();
+  element.textContent = date;
+}
```
Comment (Bondok6) [inline Modules/date time.js]: - [ ] The same thing here, kindly check my comment above ⬆

- [ ] According to the requirements, The `luxon` Package should be added as a `.js` file to the project (in full or minified version), you should download the script and save it to your project. then Import it as a module. 👍
> Download it from [here](https://moment.github.io/luxon/#/install?id=es6) 
```js
import { DateTime } from "luxon";
```

image

File: Modules/store data.js [hunk 9]
```diff
@@ -0,0 +1,6 @@
+export default function storeEntryData(array, titleElement, authorElement) {
+  const entryObject = {};
+  entryObject.title = titleElement.value;
+  entryObject.author = authorElement.value;
+  array.push(entryObject);
+}
```
Comment (Bondok6) [inline Modules/store data.js]: - [ ] The same thing here, kindly check my comment above ⬆

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @c00p75  🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No Linter errors ✅
- Correct GitHub flow ✅
- Your repo is professional ✅
- Professional README file ✅


## Required Changes ♻️

- [ ] _Check the comments under the review._ ⬇


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @c00p75,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**